backend/resolvers/queries/entryQuery.js

```diff
@@ -32,8 +32,14 @@ export function entries (_, args, req) {
     // Get all entries if no args given
     if (!args.showId && !args.studentUsername) {
       return Entry.all()
-    } else if (args.showId) { // Get entries by show
-      return Entry.findAll({ where: { showId: args.showId } })
+    } else if (args.showId) { 
+      // Get entries by show except those not excluded from judging
+      return Entry.findAll({ 
+        where: { 
+          showId: args.showId,
+          excludeFromJudging: false 
```

Comment:
hmm.... is this field nullable? if so, this might be better as "is not true" rather than "is false"

Reply:
it's not nullable and it has a default value